bug fix - outdated function duplicated from wheels internals

chips5k opened the pull request.

jkeczan commented:
We fixed the same issue in our pull request, plus we updated the code to be compatible with 1.3.4. We also did some additional development on foreign keys...check out our pull request.

https://github.com/tdm00/cfwheels-dbmigrate-plugin/pull/81

chips5k commented:
Hey - yeah I noticed that last night - after I submitted mine - sorry completely missed it but I'll switch over to yours.

Cheers - you should maybe submit your version of this to cfwheels Plugin page too - looks like this repo isn't being actively maintained by the original person? 

Sent from my iPhone

> On 28 Mar 2015, at 5:07 am, Jeremy Keczan <notifications@github.com> wrote:
> 
> We fixed the same issue in our pull request, plus we updated the code to be compatible with 1.3.4. We also did some additional development on foreign keys...check out our pull request.
> 
> #81
> 
> —
> Reply to this email directly or view it on GitHub.
>
